src/OrchardCore.Modules/OrchardCore.Media.Azure/Views/Admin/Index.cshtml

```diff
@@ -0,0 +1,20 @@
+<h1>@RenderTitleSegments(T["Azure Storage Asset Cache"])</h1>
+
+<p class="alert alert-secondary">@T["Azure Storage is configured with appsettings.json."]</p>
+
+<form asp-action="Purge">
```

Comment:
Simplified all the UI for the blob cache and moved it all to the Media.Azure Module.